src/constants/model.ts

```diff
@@ -38,10 +38,12 @@ export const Model = {
     A8: 'A8',
     Q2: 'Q2',
     Q3: 'Q3',
-    Q4: 'Q4',
+    Q4: 'Q4 e-tron',
     Q5: 'Q5',
+    Q6: 'Q6 e-tron',
```

Comment:
I think we should be explicit about these and name the key `Q4_ETRON`/`Q6_ETRON` as well.